### PR TITLE
Added IGW support, also reenabled the compatability loader.

### PR DESCRIPTION
**Pull request by MrabEzreb**
The wiki was based off of Minechem v5, even though this code is for Minechem v6. I can continue to update the wiki as more information is released for v6. (also, I am not completely sure if I am merging yours into mine, or mine into yours, so sorry if I got it wrong :P)

### CODE REVIEW

**Comment by jakimfett** [comment]
Out of curiosity...what does Ingame Wiki add that we aren't already going to do with the research journal?

**Comment by MrabEzreb** [comment]
I don't know about the research journal, but with minechem v5, it was pretty tough to figure anything out, and ur wiki didn't help much :P
also, I just really, really like IGW :)

**Comment by jakimfett** [comment]
I think maybe I'll wait on this until we get the research journal a bit more fleshed out and we can determine whether or not IGW actually adds anything. Trying to keep v6 free of duplicate or cluttered mechanics, and having two ways of gaining knowledge that need to be updated when things change starts down the path to wrong information pretty fast.

File: .gitignore [hunk 1]
```diff
@@ -1,53 +1,56 @@
-## Edits of this file MUST be in their own commit ##
```
Comment (jakimfett) [inline .gitignore]: Also, might want to re-read this line

**Comment by MrabEzreb** [comment]
alright

**Comment by MrabEzreb** [comment]
I am not completely sure where that line went, its fine on my fork :P

**Comment by jakimfett** [comment]
@MrabEzreb I was meaning more "read it, because it says changes to this file must be in their own commit".

**Comment by MrabEzreb** [comment]
OH okee :P

**Comment by jakimfett** [comment]
I'm going to go ahead and close this for now. Once we have the research journal fleshed out a bit more, if you still feel like there's a need, go ahead and jump on IRC and we can discuss it.

**Comment by MrabEzreb** [comment]
alright. The main reason that I wanted it was that the actual wiki wasn't very descriptive, so I thought, "instead of trying to fix a broken thing, why not just make v2.0?" of the wiki, not minechem, minechem is going to be v6

**Pull request closed without merging**